beater/authorization/apikey.go

```diff
@@ -69,15 +65,11 @@ func (a *apikeyAuth) IsAuthorizationConfigured() bool {
 
 // AuthorizedFor checks if the configured api key is authorized.
 // An api key is considered to be authorized when the api key has the configured privileges for the requested resource.
-// Privileges are fetched from Elasticsearch and then cached in a global cache.
-func (a *apikeyAuth) AuthorizedFor(resource string) (bool, error) {
-	if resource == "" {
-		resource = DefaultResource
-	}
-
-	//fetch from cache
-	if allowed, found := a.fromCache(resource); found {
-		return allowed, nil
+// Permissions are fetched from Elasticsearch and then cached in a global cache.
```

Comment:
Having a type `Privilege` and a type `Permission` the comments and variables should also follow this naming, as otherwise it is really confusing. That's why I asked to change the terminology consistently.